src/libs/actions/Report.js

```diff
@@ -1543,6 +1626,27 @@ function renameReport(reportID, reportName) {
         .finally(() => Onyx.set(ONYXKEYS.IS_LOADING_RENAME_POLICY_ROOM, false));
 }
 
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+    callback: (reportAction) => {
+        // // If chat report receives an action with IOU and we have an IOUReportID, update IOU object
+        // if (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && reportAction.originalMessage.IOUReportID) {
+        //     const iouReportID = reportAction.originalMessage.IOUReportID;
+
+        //     // If the IOUDetails object exists we are in the Send Money flow, and we should not fetch and update the chatReport
+        //     // as this would overwrite any existing IOUs. For all other cases we must update the chatReport with the iouReportID as
+        //     // if we don't, new IOUs would not be displayed and paid IOUs would still show as unpaid.
+        //     if (reportAction.originalMessage.IOUDetails === undefined) {
+        //         fetchIOUReportByIDAndUpdateChatReport(iouReportID, reportID);
+        //     }
+        // }
+        console.log('reportAction:', reportAction);
+
+        // TODO: get reportID from the reportAction
+        // sendLocalNotification(reportID, reportAction);
```

Comment:
This logic doesn't work because we are not getting a `reportAction` in the callback, but a list of actions

Reply:
Is the plan to call `sendLocalNotification()` on every report action? I guess we can do that - but seems like a user would get a zillion notifications for things unless we migrate all existing actions so that they are `hasAttemptedToNotify: true`. They would also get a zillion notifications when they fetch report history or anything updates a report action and we haven't yet notified for that action. Not sure this idea will work in it's current state.